src/triage/cli.py

```diff
@@ -17,9 +18,8 @@
 from triage.experiments import CONFIG_VERSION, MultiCoreExperiment, SingleThreadedExperiment
 from triage.util.db import create_engine
 
-logging.basicConfig(level=logging.INFO)
 
-import importlib.util
+logging.basicConfig(level=logging.INFO)
```

Comment:
above clean-up unrelated to initial pull request

Reply:
oh, except that that added import line could be put in a reasonable place :wink: